<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buddiesPane</class>
 <widget class="QWidget" name="buddiesPane">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout1">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="10" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>My friends</string>
     </property>
     <layout class="QGridLayout" name="gridLayout2">
      <item row="1" column="0">
       <widget class="QPushButton" name="addPB">
        <property name="toolTip">
         <string>Add a friend by its ID</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="delPB">
        <property name="toolTip">
         <string>Delete the selected friend and unsubscribe from his status</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="buddiesList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>My friends list. If empty, then try adding a few friends through the Add button.</string>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>My status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="userLabel">
          <property name="text">
           <string>Id:</string>
          </property>
          <property name="buddy">
           <cstring>userEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userEdit">
          <property name="toolTip">
           <string>My ID (in the form of username@domain)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="connectPB">
          <property name="toolTip">
           <string>Connect to server</string>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectPB">
          <property name="toolTip">
           <string>Disconnect from server and terminate any network activity</string>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Disconnected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="statusCO">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Set my published availability status</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Available</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/available.png</normaloff>:/images/available.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Away</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/away.png</normaloff>:/images/away.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Do Not Disturb</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/donotdisturb.png</normaloff>:/images/donotdisturb.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
